docs/user/getting_started.md

```diff
@@ -124,10 +123,12 @@ From the Zeta directory run the following
 ```
 $ ./src/extern/alcor-control-agent/build/build.sh
 ```
-Once ACA is done building, from the Zeta directory, you can now run a simple ping test with the command below.
+Once ACA is done building successfully, the docker image aca_build0 is created. From the zeta directory, you can now run a simple ping test with the command below.
```

Comment:
back it off, the image name is not needed from user perspective and may be changed later.